shared/middleware/helpers/property_bag.rb

```diff
@@ -30,6 +31,8 @@ def get(name)
   end
 
   def set(name, value, ip_address)
+    raise NotFound unless @can_modify
```

Comment:
@geoffrey-elliott I'd like to raise `403 Forbidden` here but I'm not sure the correct way to do that.